src/common/lib/types/protocolmessage.ts

```diff
@@ -70,13 +71,12 @@ class ProtocolMessage {
   connectionKey?: string;
   connectionSerial?: number;
```

Comment:
`ConnectionSerial` needs to be removed as per spec

Reply:
thanks will remove